flink-core/src/main/java/org/apache/flink/api/common/ArchivedExecutionConfig.java

```diff
@@ -54,6 +54,19 @@ public ArchivedExecutionConfig(ExecutionConfig ec) {
 		}
 	}
 
+	public ArchivedExecutionConfig(
```

Comment:
To change this i would have to touch the runtime Execution[[Job]Vertex] classes which i would rather not do in this PR.